pages/learn/manage/account.md

```diff
@@ -154,12 +158,20 @@ Now when you log in you will be prompted for the code displayed in your authenti
 * [Authy](https://www.authy.com)
 * [1Password](https://1password.com)
 
+## Delete account
+
+If you wish to delete your {{ $names.company.lower }} account, go to your [_Preferences_][prefs] page, and under the _Account Details_ tab, select the _Delete Account_ button. You will need to confirm this action by entering your password. Upon confirmation, the account will be permanently deleted, including all applications and devices. If you would also like to request deletion of your data in accordance with GDPR, please refer to the instructions in our [privacy policy][privacy-policy].
```

Comment:
As Alison said, we probably want to specify that if they sign up using social auth, they first need to set a password before being able to delete the account.